lib/jsonld.js

```diff
@@ -90,6 +91,11 @@ const wrapper = function(jsonld) {
 /** Registered RDF dataset parsers hashed by content-type. */
 const _rdfParsers = {};
 
+// resolved context cache
+// TODO: consider basing max on context size rather than number
+const RESOLVED_CONTEXT_CACHE_MAX_SIZE = 100;
```

Comment:
Is this adjustable?  Should there be some API so it can be adjustable or a custom cache could be used?

Reply:
It was not adjustable in the previous version, so there's no change here. We can make it so in a future PR.